mpqp/core/instruction/gates/native_gates.py

```diff
@@ -150,7 +150,7 @@ def to_other_language(
     ):
         if qiskit_parameters is None:
             qiskit_parameters = set()
-        theta = float(self.theta) if self._numeric_parameters else self.theta
+        theta = float(self.theta)
```

Comment:
theta can be float | Expr.
When we have Expr, it raises an error when converts to a float when we use to_matrix in Qiskit

Reply:
need qiskit 1.0